packages/key-did-resolver/src/index.ts

```diff
@@ -34,7 +34,7 @@ export function getResolver(): ResolverRegistry {
         didDocumentMetadata: {},
       }
       try {
-        const multicodecPubKey: any = multibase.decode(parsed.id)
+        const multicodecPubKey: any = base58btc.decode(parsed.id)
```

Comment:
PLEASE CHECK

The old multibases used this https://github.com/multiformats/js-multibase/blob/master/src/constants.js to determine the encoding.

This does not exist in multiformats. I am assuming its base58btc as in the tests the id begin with a z. Not sure if this always be the case.